src/settings.ts

```diff
@@ -304,6 +304,17 @@ export class ObsidianGitSettingsTab extends PluginSettingTab {
                     plugin.gitManager.updateGitPath(value || "git");
                 });
             });
+
+        new Setting(containerEl)
+            .setName("Base Path")
+            .addText((cb) => {
```

Comment:
A description of what `Base Path` is, is needed. Like that, it needs to be relative to the vault. In addition, I would rename it to something like `git repository path`.